docs/getting_started.md

```diff
@@ -1,57 +1,337 @@
-    For more details about Sources and Destinations please check this [section](reference/database/sources_destinations.md).
+### Create a cGroup
 
+We need a subset of runnings programs to be monitored. For this, we use the 
+Linux abstraction of [cGroups](https://www.redhat.com/sysadmin/cgroups-part-one).  
 
-<!---
-## **RAPL Formula**
+In order to create a cGroup, the following command can be used from CLI :  
 
-!!! note ""
-    for monitoring the energy consumption of your device
+```sh
+cgcreate -g perf_event:new_cgroup_name
+```
 
-RAPL Formula is made for tracking the energy consumption of your machine.
-To install RAPL Formula on a baremetal server or a PC run [the following
-script](script/rapl_install.sh) in a Terminal.
+Check [here](./reference/cgroup/cgroup_v1_activation.md) if you have trouble 
+creating the cgroup.  
 
-The script explains what it will do and then pauses before it does it.
+### Add programs to the group
 
-Please notice that you need a **Linux distribution** in order to use the HWPC Sensor installed by the script as
-well as a **comptible Intel** (Sandy Bridge and newer) or **AMD Processor** (Zen). **Power/ARM/RISCV are not supported** architectures. HWPC Sensor will **not work on a Virtual Machine**. However, you can install the Formula by hand in a Virtual Machine if need it.
--->
+Once the group created, we need to fill it with programs to be monitored. 
+To do so, you can use the following :  
 
-## **SmartWatts Formula**
+```sh
+cgclassify -g perf_event:new_cgroup_name PID
+```
 
-!!! note ""
-    for monitoring the power consumption of your process
+with `PID`, the pid of the process you want to monitor. If you want to monitor a
+program composed of many process, replace PID with `$(pidof program_name)`.
 
-Smartwatts is made for tracking the power consumption of processes on a
-machine.
-To install Smartwatts on a baremetal server or a PC run [the following
-script](script/smartwatts_install.sh) in a Terminal. Please notice that you will need [pip](https://pip.pypa.io/en/stable/installation/) or [docker](https://docs.docker.com/engine/install/) in order to use the Formula.
+### Example program to monitor
 
-The script explains what it will do and then pauses before it does it.
+[stress-ng](https://wiki.ubuntu.com/Kernel/Reference/stress-ng) can be used to 
+generate load on one's system.  
+An example usage, once installed :  
 
-Please notice that you need a **Linux distribution** in order to use the HWPC Sensor installed by the script as
-well as a **comptible Intel** (Sandy Bridge and newer) or **AMD Processor** (Zen). You also need [docker](https://docs.docker.com/engine/install/).  **Power/ARM/RISCV are not supported** architectures. HWPC Sensor will **not work on a Virtual Machine**. However, you can install the Formula by hand in a Virtual Machine if need it.
+```sh
+stress-ng --cpu 1 --timeout 5m
+```
 
+The PID can be found using :  
 
+```sh
+STRESS_PID="$(pgrep stress-ng)"
+```
```

Comment:
Here we are maybe missing the complete example for running the program, creating the cgroup and associating the program to the cgroup